packages/components/src/internal/components/pipeline/actions.ts

```diff
@@ -0,0 +1,30 @@
+import { ActionURL, Ajax, Utils } from "@labkey/api";
+import { PipelineStatusDetailModel } from "./model";
+import { resolveErrorMessage } from "../../..";
+
+export function getPipelineStatusDetail(rowId: number, offset?: number, count?: number ): Promise<PipelineStatusDetailModel> {
+    return new Promise((resolve, reject) => {
+        Ajax.request({
+            url: ActionURL.buildURL('pipeline-status', 'statusDetails.api'),
+            method: 'GET',
+            params: {
+                rowId,
+                offset,
+                count
+            },
+            success: Utils.getCallbackWrapper(response => {
+                if (response.success) {
+
+                    resolve(PipelineStatusDetailModel.loadResult(response.data));
+                } else {
+                    console.error(response);
+                    reject('There was a problem retrieving the pipeline status detail.');
```

Comment:
Hmm.  "Pipeline" is not a term users will be familiar with, I think.  Perhaps just use "the details." or "the status details."